keras/backend/tensorflow_backend.py

```diff
@@ -113,6 +113,32 @@ def _to_tensor(x, dtype):
     return x
 
 
+def is_sparse(tensor):
+    return isinstance(tensor, tf.SparseTensor)
+
+
+def how_sparse(tensors):
```

Comment:
Unnecessary. Use `all([K.is_sparse(x) for x in tensors])` or `any([K.is_sparse(x) for x in tensors])`. Far more readable than an obscure API call for which you would have to read the source to understand what it does.